doc/src/manual/parallel-computing.md

```diff
@@ -1511,8 +1511,8 @@ For instance functions that have their
 [name ending with `!`](https://docs.julialang.org/en/latest/manual/style-guide/#Append-!-to-names-of-functions-that-modify-their-arguments-1)
 by convention modify their arguments and thus are not pure. However, there are
 functions that have side effects and their name does not end with `!`. For
-instance [`findfirst(regex, str)`](@Ref) mutates its `regex` argument or
-[`rand()`](@Ref) changes `Base.GLOBAL_RNG` :
+instance [`findfirst`](@ref) mutates its `regex` argument or
```

Comment:
Perhaps keep `(regex, str)`, otherwise what follows does not make any sense.

Reply:
fixed; I was afraid that it was also breaking a link (but I understand it does not).